backoffice/templates/backoffice/users/details.html

```diff
@@ -0,0 +1,73 @@
+{% extends "backoffice_base.html" %}
+
+{% block heading %}<div class="text-lg font-medium leading-6 text-gray-900 sm:truncate">User - {{ user_details.username }} ({{ user_details.first_name|title|add:" "|add:user_details.last_name|title }})</div>{% endblock %}
+
+
+{% block main %}
+<div class="p-4 sm:p-6 lg:p-8">
+    <div>
+        <h3 class="text-xl font-semibold text-gray-900">Bookings</h3>
+
+        <div class="mt-8 flex flex-col">
+            <div class="-my-2 -mx-4 overflow-x-auto sm:-mx-6 lg:-mx-8">
+                <div class="inline-block min-w-full py-2 align-middle md:px-6 lg:px-8">
+                    <div class="overflow-hidden shadow ring-1 ring-black ring-opacity-5 md:rounded-lg">
+                        {% include "backoffice/bookings/bookings_list.html" with bookings=page %}
+                    </div>
+                </div>
+            </div>
+            <div class="mt-1">
+                {% include "backoffice/components/paginator.html" with page=page page_range=page_range %}
+            </div>
+        </div>
+    </div>
+    <div class="px-5 py-2">
```

Comment:
I think this section should be at the top above bookings, but open to discussion if you disagree.